### PR TITLE
Change pom.xml groupId and version, reformat project tag

### PR DESCRIPTION
**Pull request by nxtk**
Change pom.xml groupId and version, reformat project tag

### CODE REVIEW

**Comment by nxtk** [comment]
Probably must change project package to ./src/main/java/com.github/BoscoJared/disparse/

File: pom.xml [hunk 1]
```diff
@@ -1,9 +1,11 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (BoscoJared) [inline pom.xml]: I would make sure we are consistent with styling, spacing, etc.

I'm using google-style for disparse, and that hasn't formatted things like this multi-lined

File: pom.xml [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
     <modelVersion>4.0.0</modelVersion>
-    <groupId>disparse</groupId>
+    <groupId>com.github.BoscoJared</groupId>
```
Comment (BoscoJared) [inline pom.xml]: Not 100% crazy with this change as reverse domain is really nice when you are an entity i.e. Facebook has com.facebook.* but for a small open-source project having my username / full name in every package seems undesirable.

File: src/main/java/com/github/boscojared/disparse/discord/Dispatcher.java [hunk 3]
```diff
@@ -51,38 +52,54 @@ public void optionRequired(MessageReceivedEvent event, String message) {
...
-                .sorted(
-                        Comparator.comparing((Flag flag) -> toLower(flag.getShortName()), Comparator.nullsLast(Comparator.naturalOrder()))
-                            .thenComparing(flag -> flag.getLongName().toLowerCase())
-                )
-                .collect(Collectors.toList());
 
-        List<Command> subcommands = commands.stream()
+        List<CommandFlag> sortedFlags = flags.stream().sorted(helpCommandFlagComparator).collect(Collectors.toList());
+
+        List<Command> subCommands = commands.stream()
```
Comment (BoscoJared) [inline src/main/java/com/github/boscojared/disparse/discord/Dispatcher.java]: subcommand is one word so I don't think camelCase is necessary here

File: src/main/java/com/github/boscojared/disparse/parser/Parser.java [hunk 4]
```diff
@@ -1,28 +1,34 @@
...
-    public Parser(final Map<Command, ? extends Collection<Flag>> commandToFlags) {
+    public Parser(final Map<Command, ? extends Collection<CommandFlag>> commandToFlags) {
         this.commandToFlags = commandToFlags;
     }
 
     public ParsedOutput parse(List<String> args) {
         final Command commandName = this.findCommand(args);
 
-        this.createShortAndLongOptions(commandName);
+        fillShortAndLongOptions(commandName);
```
Comment (BoscoJared) [inline src/main/java/com/github/boscojared/disparse/parser/Parser.java]: Just for consistency use `this.fillShortAndLongOptions` I pretty much always use `this` in this situation, and having it mixed-and-matched all over won't be great

File: src/main/java/com/github/boscojared/disparse/parser/reflection/Detector.java [hunk 5]
```diff
@@ -38,24 +39,26 @@ public void reportMethodAnnotation(Class<? extends Annotation> annotation,
                                     CommandHandler handler =
                                             method.getAnnotation(CommandHandler.class);
                                     Command command = new Command(handler.commandName(),
-                                            handler.description(), handler.roles());
+                                                                  handler.description(), handler.roles());
```
Comment (BoscoJared) [inline src/main/java/com/github/boscojared/disparse/parser/reflection/Detector.java]: I feel like formatted with google-style this won't have such a large space here but I could be wrong

File: src/main/java/com/github/boscojared/disparse/parser/reflection/Detector.java [hunk 6]
```diff
@@ -38,24 +39,26 @@ public void reportMethodAnnotation(Class<? extends Annotation> annotation,
...
                                             for (Field field : fields) {
                                                 if (field.isAnnotationPresent(Flag.class)) {
-                                                    disparse.parser.Flag parseableFlag = Utils
+                                                    CommandFlag flag = Utils
                                                             .createFlagFromAnnotation(field, field
                                                                     .getAnnotation(Flag.class));
-                                                    CommandRegistrar.registrar.register(command,
-                                                            parseableFlag);
+                                                    CommandRegistrar.REGISTRAR
+                                                        .register(command,
```
Comment (BoscoJared) [inline src/main/java/com/github/boscojared/disparse/parser/reflection/Detector.java]: similarly here with google-style; again, I could just be wrong and this is how it formatted it

File: src/main/java/com/github/boscojared/disparse/discord/Dispatcher.java (hunk 3 above)
Comment (nxtk) [reply src/main/java/com/github/boscojared/disparse/discord/Dispatcher.java]: agree, googled sub-command for some reason

File: pom.xml (hunk 2 above)
Comment (nxtk) [reply pom.xml]: alright, fair enough

File: src/main/java/com/github/boscojared/disparse/parser/Parser.java (hunk 4 above)
Comment (nxtk) [reply src/main/java/com/github/boscojared/disparse/parser/Parser.java]: right, i never seen this used for methods in constructors

File: src/main/java/com/github/boscojared/disparse/parser/reflection/Detector.java (hunk 5 above)
Comment (nxtk) [reply src/main/java/com/github/boscojared/disparse/parser/reflection/Detector.java]: right i used tweaked google style from another project, needs reformating

**Pull request merged**